easybuild/easyconfigs/m/makedepf90/makedepf90-2.8.8-foss-2017a.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'makedepf90'
+version = '2.8.8'
+
+homepage = ''
```

Comment:
@dplacencia How about using `https://linux.die.net/man/1/makedepf90` instead? Better than leaving it empty imho...